src/components/sockend.js

```diff
@@ -84,31 +84,45 @@ module.exports = class Sockend extends Configurable(Component) {
             publisherNamespaces['/' + normalizedNamespace] = true;
             obj.namespace = namespace;
 
+            const broadcasts = new Set(obj.advertisement.broadcasts);
```

Comment:
There's no need for this functionality — you can remove everything related to `broadcasts`, i.e., line 119.